.github/workflows/deploy.yml

```diff
@@ -8,12 +8,56 @@ on:
     branches: [ master ]
 
 jobs:
-  deploy:
-    name: Deploy Demo App
+  build_browser_demo:
+    name: Build Browser Demo
+    runs-on: ubuntu-latest
+    env:
```

Comment:
For `build_browser_demo` we dont need this section and this AWS configuring should go into each deploy job?
```
env:
        AWS_DEFAULT_REGION: us-east-1
        AWS_DEFAULT_OUTPUT: text
        AWS_ACCESS_KEY_ID: ${{secrets.AWS_ACCESS_KEY}}
        AWS_SECRET_ACCESS_KEY: ${{secrets.AWS_SECRET_ACCESS_KEY}}
    steps:
      - name: Configure AWS Credentials
        uses: aws-actions/configure-aws-credentials@v1
        with:
          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY }}
          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
          aws-region: us-east-1
      - name: Checkout Package
        uses: actions/checkout@v2
        with:
          fetch-depth: 0
      - name: Install SAM CLI
        run: script/github-action-awscli-installation
```

Reply:
Yeah, each job runs in its own container with no context from the other jobs, so you can't build it once here and expect the other jobs to have the meeting and meeting readiness checker built. 

If you want to build the demo before deploying them, you'll have to build them in each job seperately. Else, you can do `npm run build` within the deploy script instead of having it in the github action. That might be better.

Reply:
Tested we do not need to run "npm run build" deploy script builds the app before deploying